foxai/explainer/computer_vision/algorithm/gradient_shap.py

```diff
@@ -165,39 +173,46 @@ def calculate_features(
         """
         gradient_shap = self.create_explainer(model=model, layer=layer)
 
-        # defining baseline distribution of images
-        if baselines is None:
-            baselines = torch.randn(
-                (
-                    2 * input_data.shape[0],
-                    *input_data.shape[1:],
-                ),
-                requires_grad=True,
-                device=input_data.device,
-            )
-
-        if isinstance(gradient_shap, LayerGradientShap):
-            attributions = gradient_shap.attribute(
-                input_data,
-                n_samples=n_samples,
-                stdevs=stdevs,
-                baselines=baselines,
-                target=pred_label_idx,
-                return_convergence_delta=False,
-                additional_forward_args=additional_forward_args,
-                attribute_to_layer_input=attribute_to_layer_input,
-            )
-        else:
-            attributions = gradient_shap.attribute(
-                input_data,
-                n_samples=n_samples,
-                stdevs=stdevs,
-                baselines=baselines,
-                target=pred_label_idx,
-                return_convergence_delta=False,
-                additional_forward_args=additional_forward_args,
-            )
-        validate_result(attributions=attributions)
+        attributions_list: List[torch.Tensor] = []
```

Comment:
And same here.

Reply:
Fixed: https://github.com/softwaremill/FoXAI/pull/97/commits/e62e2b2b737447cb080e63aedf5847054f1d3b1d.